handler.py

```diff
@@ -14,38 +16,120 @@
 SQS_CLIENT = boto3.client('sqs')
 
 
-def runScheduledObservatoryScan(event, context):
+def addPortScanToQueue(event, context):
+    data = json.loads(event['body'])
```

Comment:
Are there  de-serialization vectors we should consider for JSON.loads (similar to YAML.load) and is there an equivalent to Rubies YAML.safe_load we should consider?  Maybe this isn't a thing, but wanted  to raise for the sake of safety.

Reply:
There is a `yaml.safe_load()` in Python as well, however json loading with `json.loads()` appears to be safe (based on Internet perusing about this subject).

Reply:
ok